src/main/java/ca/ryangreen/apigateway/generic/GenericApiGatewayResponse.java

```diff
@@ -11,7 +11,11 @@
 
     public GenericApiGatewayResponse(HttpResponse httpResponse) throws IOException {
         this.httpResponse = httpResponse;
-        this.body = IOUtils.toString(httpResponse.getContent());
+        if (httpResponse.getContent()!= null) {
+            this.body = IOUtils.toString(httpResponse.getContent());
+        } else {
+            this.body = null;
```

Comment:
Is this else block required? Isn't body null by default?

Reply:
because it is final, there are "Error:(17, 5) java: variable body might not have been initialized" without it